lua/spectre/search/base.lua

```diff
@@ -59,7 +95,7 @@ base.search = function(self, query)
 
     -- no more args
     table.insert(args, "--")
-    args = utils.tbl_remove_dup(flatten(args))
```

Comment:
Any specific reason you were removing dupes from the args? If you do that, multiple paths don't work with ripgrep, because you have to supply multiple `-g` flags.

Reply:
I don't remember it now but I think we can remove it.